### ISSUE TITLE
docs - Which accessibility standard is Bootstrap 4 following? WCAG 2.0 A, AA or AAA?

### ISSUE THREAD

**Issue by changLiuUNSW** [twbs/bootstrap#22406]
We are using Bootstrap 4 to build public facing application and the key requirement is that the site must meet WCAG 2.0 AA compliant.
Just want to know which accessibility standard is Bootstrap 4 following. WCAG 2.0 A, AA or AAA?
It is better to clarify it in the documentation.
Thanks

**Comment by bkdotcom** [comment]
https://www.w3.org/TR/WCAG20/#guidelines

> 1 Perceivable
> 
> 1. Provide text alternatives for any non-text content so that it can be changed into other forms people need, such as large print, braille, speech, symbols or simpler language.
> 2. Provide alternatives for time-based media.
> 3. Create content that can be presented in different ways (for example simpler layout) without losing information or structure.
> 4. Make it easier for users to see and hear content including separating foreground from background.
> 
> 2 Operable
> 
> 1. [Make all functionality available from a keyboard.](/twbs/bootstrap/issues/22402) 😃
> 2. Provide users enough time to read and use content.
> 3. Do not design content in a way that is known to cause seizures.
> 4. Provide ways to help users navigate, find content, and determine where they are.
> 
> 3 Understandable
> 
> 1. Make text content readable and understandable.
> 2. Make Web pages appear and operate in predictable ways.
> 3. Help users avoid and correct mistakes.
> 
> 4 Robust
> 
> 1. Maximize compatibility with current and future user agents, including assistive technologies.

Bootstrap is a frame work.  It makes it possible to create compliant websites, but compliance is still up to the developer.  It also allows you to create a website that is very much not in compliance.

**Comment by patrickhlauke** [comment]
we strive towards making interactive controls/widgets expose all the correct roles/values/state properties (to satisfy, say, 4.1.2), to make things works with keyboard (2.1.1). Some of the default colours / their combinations do not meet color contrast requirements (so would fail 1.4.3 / AA).

but beyond that, as @bkdotcom noted, this is simply a framework. a set of tools. still up to the developer to make sure what they're building WITH it meets accessibility requirements. but for the most part (barring bugs of course), it should be at least feasible to use vanilla bootstrap and (provided you know what you're doing) build something A or even AA (depending on use of colours) compliant.

**Comment by changLiuUNSW** [comment]
@patrickhlauke 
I totally agree that Bootstrap is a framework and developers can customize it to meet A or AA.
However, I still believe it is better to clarify accessibility standard/level in the documentation, because some companies take accessibility very seriously when choosing third party frameworks. 
Take my company (Bank) for example, we did a lot of discussion whether we can use Bootstrap or not, but some people still think Bootstrap is not accessibility like early Bootstrap 2 and 3.
We did a lot of efforts to tell/demonstrate how good the Bootstrap accessibility is, but they still have some concerns about it.
I really appreciate all the efforts you guys did for accessibility in Bootstrap 4 and it would be big help for us if you guys can clarify accessibility standard/level in the documentation.
